cdc/redo/writer/file.go

```diff
@@ -568,5 +584,18 @@ func (w *Writer) writeToS3(ctx context.Context, name string) error {
 	}
 
 	// Key in s3: aws.String(rs.options.Prefix + name), prefix should be changefeed name
-	return cerror.WrapError(cerror.ErrS3StorageAPI, w.storage.WriteFile(ctx, filepath.Base(name), fileData))
+	err = w.storage.WriteFile(ctx, filepath.Base(name), fileData)
+	if err != nil {
+		return cerror.WrapError(cerror.ErrS3StorageAPI, err)
+	}
+
+	// in case the page cache piling up triggered the OS memory reclaming which may cause
+	// I/O latency spike, we mandatorily drop the page cache of the file when it is successfully
+	// written to S3.
+	err = fsutil.DropPageCache(name)
```

Comment:
Is there a way to put this line in `fileAllocator`?

Reply:
I think the page cache evicting is irrelevant with the file allocation.